core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -56,6 +56,7 @@ def install_file_attachment
       if options[:active_storage]
         say "Installing Active Storage", :green
         rake 'active_storage:install'
+        append_file "config/initializers/spree.rb", "Rails.application.config.active_storage.variant_processor = :mini_magick"
```

Comment:
Hey, @peterberkenbosch, that could cause a conflict with the user's preference, right? We should support both by checking the configured value